che-terminal-client/pom.xml

```diff
@@ -57,42 +57,45 @@
                 </configuration>
             </plugin>
             <plugin>
-                <artifactId>maven-antrun-plugin</artifactId>
+                <groupId>com.github.eirslett</groupId>
+                <artifactId>frontend-maven-plugin</artifactId>
                 <executions>
                     <execution>
-                        <!--clean-up build folders-->
-                        <id>clean-up</id>
-                        <phase>clean</phase>
+                        <id>install node and npm</id>
+                        <phase>generate-resources</phase>
                         <goals>
-                            <goal>run</goal>
+                            <goal>install-node-and-npm</goal>
                         </goals>
                         <configuration>
-                            <tasks>
-                                <delete dir="${basedir}/node_modules" />
-                                <delete dir="${basedir}/build" />
-                            </tasks>
+                            <arguments>install --fetch-retry-mintimeout 60000 --fetch-retries 10 --maxsockets 2</arguments>
```

Comment:
should args should be properties, so it's easy to override them by user ? and also reuse these properties elsewhere

Reply:
sure, good idea.